src/main/java/org/jenkinsci/extension_indexer/SourceAndLibs.java

```diff
@@ -117,34 +120,35 @@ public List<String> getViewFiles(String pkg) {
 
     public static SourceAndLibs create(Module module) throws IOException, InterruptedException {
         final File tempDir = File.createTempFile("jenkins","extPoint");
-        tempDir.delete();
-        tempDir.mkdirs();
+        Files.delete(tempDir.toPath());
+        Files.createDirectories(tempDir.toPath());
```

Comment:
If what you're looking for is a temporary directory, `Files#createTempDirectory` is a more straightforward path to get there and avoids a race condition that I think Jonathan Leitschuh has been working on fixing throughout the entire planet with automated PRs.

Reply:
Thanks!  Included in 4f8dd3b226130f0a7a2d2272cf4731d21ccb3957

Reply:
We fixed the temporary directory creation issue, but I'm not sure whether this repository also contains a zip slip issue. @JLLeitschuh are you interested in running your tool against it and creating a PR if there is a zip slip issue?

Reply:
> I think Jonathan Leitschuh has been working on fixing throughout the entire planet with automated PRs.

I have been, this is a hilarious way of phrasing this one 😆 ❤️

> We fixed the temporary directory creation issue, but I'm not sure whether this repository also contains a zip slip issue. @JLLeitschuh are you interested in running your tool against it and creating a PR if there is a zip slip issue?

If you could do me a kind favor and get any projects (ie probably this entire org) you do want me to generate PRs against indexed into this project, I'd be more than happy to do so! 🎉 

https://github.com/moderneinc/jenkins-ingest

Reply:
Sorry, I do not understanding what you are asking me to do. I just thought you might be interested in contributing another helpful automated PR, as you have in the past. If you are not, no problem. We appreciate your past contributions!